types/atom/index.d.ts

```diff
@@ -2403,27 +2402,25 @@ export interface TextEditorRegistry {
     observe(callback: (editor: TextEditor) => void): Disposable;
 }
 
+export type TooltipPlacement =
+    |"top"|"bottom"|"left"|"right"
+    |"auto"|"auto top"|"auto bottom"|"auto left"|"auto right";
+
 /** Associates tooltips with HTML elements or selectors. */
 export interface TooltipManager {
     /** Add a tooltip to the given element. */
     add(target: HTMLElement, options: {
-        title?: string,
-        html?: boolean,
-        item?: HTMLElement|{ element: HTMLElement },
-        class?: string,
-        placement?: "top"|"bottom"|"left"|"right"|"auto"|(() => string),
-        trigger?: "click"|"hover"|"focus"|"manual",
-        delay?: { show: number, hide: number },
-        keyBindingCommand?: string,
-        keyBindingTarget?: HTMLElement
+        item?: HTMLElement|{ element: HTMLElement }
```

Comment:
Hmm. I just noticed this was left unchanged. I have to wonder if it's intended or an oversight.

I've mentioned this is seems too strict in passing, but maybe didn't emphasize it enough?

> Also while on topic, I think item can be anything that Atom's view algorithm will get a view for. At least it's usage suggests that: https://github.com/atom/atom/blob/v1.22.1/src/tooltip.js#L342

So... it probably should just be `object`?

Reply:
It should be. I had read it, but probably just associated it with the other `item` change in the decoration options.